mmdet/models/losses/eqlv2_loss.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+from functools import partial
+from typing import Optional
+
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+import torch.nn.functional as F
+from mmengine.logging import print_log
+from torch import Tensor
+
+from mmdet.registry import MODELS
+
+
+@MODELS.register_module()
+class EQLV2Loss(nn.Module):
```

Comment:
Please add a unit test

Reply:
Is there an algorithm in MMDetection that uses this algorithm, I need to verify the correctness

Reply:
> Is there an algorithm in MMDetection that uses this algorithm, I need to verify the correctness

Got it, I will work on the unit test. Meanwhile, I don't think there is any algorithm in MMDet uses this one.